### ISSUE TITLE
Deselecting single element after selecting all elements

### ISSUE THREAD

**Issue by yatki** [excalidraw/excalidraw#2765]
Hey there, 

Brilliant work so far, thank you very much for the tool.

Currently deselecting individual elements after selecting all elements is not possible. So after selecting all elements, when I hold cmd key (or could be ctrl etc.) and click on any elements, I'd expect those elements to be deselected.

Would you consider adding this feature?

Have a great week,
Cheers,

**Comment by dwelle** [comment]
Hello,

<kbd>Cmd</kbd> is for selecting single element (among other things). What you're looking for is <kbd>Shift</kbd>.

We should maybe reword the shortcut hint to `Add/remove element from selection` or something...

![image](https://user-images.githubusercontent.com/5153846/104166664-6ae19f00-53fb-11eb-8383-90e98f9f53e0.png)
